src/locales/en/common.json

```diff
@@ -488,7 +488,8 @@
       "invalidSession": "Invalid Wallet Connect session",
       "alreadyPending": "A connection is already waiting"
     },
-    "reverted": "reverted"
+    "reverted": "reverted",
```

Comment:
I know you didn't change this line but - just wondering how we might humanise this term "reverted"? I have horrible memories of this term when building out PPN!

Reply:
This translation here is hooked with Smart Wallet deployment errors, it's generic error that may be returned from Archanova, but I think we applied it as translatable and it's returned when Smart Wallet account deployment estimate method returns error. However, it is also used as string constant to check the error so it actually can either be split as translation and as a constant separately and translation changed to something more readable or we can leave as it is.

Reply:
I personally would split it - the justification is that these translation files can, at any time, be changed remotely. A public translator may innocently come along and translate the term "reverted" - which would result the string constant check breaking.